cannon/mipsevm/evm_test.go

```diff
@@ -76,13 +76,13 @@ func (m *MIPSEVM) Step(t *testing.T, stepWitness *StepWitness) []byte {
 
 	if stepWitness.HasPreimage() {
 		t.Logf("reading preimage key %x at offset %d", stepWitness.PreimageKey, stepWitness.PreimageOffset)
-		poInput, err := stepWitness.EncodePreimageOracleInput()
+		poInput, err := stepWitness.EncodePreimageOracleInput(0)
```

Comment:
Should local context be its own type? What does 0 mean? ie make it human readable

Reply:
I've removed these in the PR above - #7630. They're not consumed by anything else other than the tests, so it's just wasted space on disk.

Will add a custom type for the tests.